### ISSUE TITLE
Make Auth UI Signup / Login forms more customizable

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#2174]
- allow changing the order of the default fields
- allow customizing text
- allow adding additional text

**Comment by Martinsos** [comment]
Related: https://github.com/wasp-lang/wasp/discussions/2171 .

**Comment by infomiho** [comment]
> allow customizing text

We could handle this in #2061

**Comment by Martinsos** [comment]
> > allow customizing text
> 
> We could handle this in #2061

True, although here we could do it more directly and not wait for that one! I think we probably want eevn more flexibility here.

**Comment by infomiho** [comment]
+ styling of OAuth buttons: https://ptb.discord.com/channels/686873244791210014/1294954558337847328/1294954558337847328

**Comment by FranjoMindek** [comment]
Duplicate of https://github.com/wasp-lang/wasp/issues/1617, forgot to comment.
